_historical_review/core-tasks/app/services/redis_cache_service.py

```
@@ -0,0 +1,447 @@
+            # Cache hit
+            if self.metrics:
+                self.metrics.hits += 1
+
+            # Refresh TTL if requested
+            if refresh_ttl and hasattr(self, 'refresh_ttl'):
+                self.redis_client.expire(key, self.config.ttl_seconds)
+
+            # Deserialize
+            result = json.loads(cached.decode('utf-8'))
+            logger.debug(f"Cache hit: {key}")
+            return result
```

Ruff (0.14.13)

Consider moving this statement to an `else` block

(TRY300)